packages/commons/src/logger.ts

```diff
@@ -11,7 +11,19 @@
 import pino from 'pino'
 export const logger =
   process.env.NODE_ENV === 'production'
-    ? pino()
+    ? pino({
+        redact: [
+          'req.headers.authorization',
+          'req.remoteAddress',
+          "req.headers['x-real-ip']",
+          "req.headers['user-agent']",
+          "req.headers['x-real-user-agent']",
+          "req.headers['traceparent']",
+          "req.headers['tracestate']",
+          "req.headers['elastic-apm-traceparent']",
```

Comment:
```suggestion
```

Good point. I don't think user-agent should be removed either. There's a balance between being able to debug and get information from the logs and not exposing anything that can endanger the user's security / privacy.